app/src/main/java/com/appunite/loudius/network/utils/AuthFailureHandler.kt

```diff
@@ -20,6 +20,7 @@ import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.SharedFlow
 import javax.inject.Singleton
 
+// Question: Do we need this interface if there is only one implementation of it?
 interface AuthFailureHandler {
     suspend fun emitAuthFailure()
```

Comment:
Nicely spotted. It's a leftover, firstly I started with a fake implementation and then I decided to use mock, but the interface left.